robolectric-processor/src/test/java/org/robolectric/annotation/processing/RobolectricProcessorTest.java

```diff
@@ -32,6 +33,12 @@ public void roboProcessor_supportsPackageOption() {
   }
   
   @Test
+  public void roboProcessor_supportsShouldInstrumentPackageOption() {
+    ASSERT.that(
```

Comment:
I'm following other tests for consistency. Maybe we can update the dependency and import in the future? There should be more usages of org.truth0 in other tests.